stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -97,12 +97,21 @@ public func _stdlib_select(
         let readAddr = readfds.baseAddress
         let writeAddr = writefds.baseAddress
         let errorAddr = errorfds.baseAddress
+#if CYGWIN
+        func asFdSetPtr(
+          _ p: UnsafeMutablePointer<UInt>?
+        ) -> UnsafeMutablePointer<_types_fd_set>? {
```

Comment:
Updating to 7/29 snapshot, I added new patches.

BTW, I don't know why this is differ to Linux.
The definition part in `sys/select.h` is as follows.
```
typedef struct _types_fd_set {
    fd_mask fds_bits[_howmany(FD_SETSIZE, NFDBITS)];
} _types_fd_set;

#define fd_set _types_fd_set
```
`fd_set` did not work, but `_types_fd_set` worked.